test/helpers/kubectl.go

```diff
@@ -396,6 +396,56 @@ func (kub *Kubectl) MicroscopeStart() (error, func() error) {
 	return nil, cb
 }
 
+// TcpdumpStart runs tcpdump on all tcpdumps pods and return a callback
+// function to close the tcpdump and store the logs in test result folder
+func (kub *Kubectl) TcpdumpStart() (error, func() error) {
+	var tcpDumpCmd = "%s -n -i any not port 22 -w %s"
+	var containerVolumeMount = "/test/"
+	var cb = func() error { return nil }
+
+	testPath, err := CreateReportDirectory()
+	if err != nil {
+		kub.logger.WithError(err).Errorf(
+			"cannot create test results path '%s'", testPath)
+		return err, cb
+	}
+
+	_ = kub.Apply(ManifestGet(TcpdumpManifest))
```

Comment:
Why don't we check for errors here?

Reply:
because Tcpdump pod is never deleted. we only need to make sure that is in place.